metriq/models/method.py

```diff
@@ -10,52 +12,29 @@ class Method(TeaClientModel):
 
     Attributes:
         id (str): Method ID.
-        userId: ID of user who submitted method.
         name (str): Method short name.
-        fullName (str): Method full name.
+        full_name (str): Method full name.
```

Comment:
I believe this was right as `camelCase` rather than `snake_case`. Can you please check, and revert this line if so?

Reply:
By the way, just say "Done" on these "conversations" and resolve them, if they're done, always. I'll do it, just this time.